### DISCUSSION TITLE
Modern List Form Customisation - PnP Support?

### DISCUSSION THREAD

**Discussion by anthonywhite** [pnp/pnpframework#395] [Q&A] [answered]
Is there any way now (or planned) to support the simpler non-JSON modern list form customisations - showing/hiding fields (conditionally or not) - in PnP templates?

**Comment by jansenbe** [comment, marked as answer]
Currently we don't have specific support for that in the PnP templates, but please add this as suggestion in here https://github.com/pnp/PnP-Provisioning-Schema/issues so that in can be evaluated when we design the next version of the PnP provisioning template schema.
